test/e2e/federated-service.go

```diff
@@ -122,6 +126,30 @@ var _ = framework.KubeDescribe("[Feature:Federation]", func() {
 				framework.SkipUnlessFederated(f.ClientSet)
 
 				nsName := f.FederationNamespace.Name
+				// Create kube-dns configmap for kube-dns to accept federation queries.
+				federationDomain := os.Getenv("DNS_ZONE_NAME")
```

Comment:
Please use `FEDERATIONS_DOMAIN_MAP`. See https://github.com/kubernetes/test-infra/blob/master/jobs/ci-kubernetes-e2e-gce-federation.sh#L39. This already has the string in the appropriate format for the `kube-dns`'s `--federations` flag/key.